src/fluree/server/components/consensus.clj

```diff
@@ -11,10 +11,10 @@
   (log/debug "Starting raft consensus with config:" config)
   (let [handler (consensus-handler/create-handler consensus-handler/default-routes)]
     (consensus/start handler (assoc config :join? false
```

Comment:
i'm not sure if cljfmt can handle this, but i think multi-line assoc calls should have their first key-value pair start on a new line so that all the k-v pairs line up for easy scanning. here, the `:join? false` is on a different column from all the other pairs, so someone doing a quick scan to see which attributes are set to one could easily miss it.

Reply:
Yeah, I don't _think_ it can enforce that.